### PR TITLE
Templar Helmets Sprites + New Malum helmet

### PR DESCRIPTION
**Pull request by Gyrandola**
## About The Pull Request

Ports Astrata, Necra and Malum helmets from https://github.com/GeneralPantsuIsBadAtCoding/Azure-Peak/pull/513

Malum is completely new, and it has also been added as a recipe.

We already have an astratan and necran one, so I added them as **alternative versions** because they are both cool.

| Helm  | Sprite |
| ------------- | ------------- |
| Astrata  |  374326180-1e319c0d-ca47-4b7a-936f-5f8deb2deaad image|
| Necra | 374325398-2c1f5d0b-e22e-4bc5-bc88-79652c3180ba image|
| Malum| 374325552-546f8a03-aec4-4c93-be21-0b6494f0b6ad image|

### CODE REVIEW

**Comment by Nakalon** [comment]
This is very sexy

**Comment by SayonaraEcho** [comment]
Hot

**Comment by radarseas** [comment]
any chance of making these varients either right click swappable a la, shield heraldry or forgable unlike there?

its gonna really suck when your exclusive helmet breaks and you can't repair nor fix it.

**Comment by Gyrandola** [comment]
> any chance of making these varients either right click swappable a la, shield heraldry or forgable unlike there?

Right now its a 50 50 to spawn with a variant or the other, Its not hard to make it right clickable but I've got other stuff on my plate right now, someone else can easily do it.

> its gonna really suck when your exclusive helmet breaks and you can't repair nor fix it.

They are forgeable

**Comment by katsukai** [comment]
Cool sprites.

File: code/modules/clothing/rogueclothes/hats.dm [hunk 1]
```diff
@@ -576,8 +576,23 @@
...
+
+/obj/item/clothing/head/roguetown/helmet/heavy/malumhelm
+	name = "malummite helmet"
+	desc = "Forged in a coal-black, this helmet carries a sigiled blade upon it's visor, ever reminding it's wearer of Malum's powerful gaze."
+	icon_state = "malumhelm"
+	item_state = "malumhelm"
+	emote_environment = 3
+	flags_inv = HIDEEARS|HIDEFACE|HIDEHAIR
+	block2add = FOV_RIGHT|FOV_LEFT
+	smeltresult = /obj/item/ingot/steel
```
Comment (Nakalon) [inline code/modules/clothing/rogueclothes/hats.dm:592]: I know im pulling a 🤓 ☝️  But smelt result is already defined in /obj/item/clothing/head/roguetown/helmet
as steel

image

Comment (Nakalon) [reply code/modules/clothing/rogueclothes/hats.dm:592]: armor_class is as well in helmet/heavy

image

**Review by Nakalon — CHANGES_REQUESTED** [review]

File: code/modules/clothing/rogueclothes/hats.dm (hunk 1 above)
Comment (Gyrandola) [reply code/modules/clothing/rogueclothes/hats.dm:592]: > armor_class is as well in helmet/heavy
> 
> image

the other helmets are not heavy

Comment (Nakalon) [reply code/modules/clothing/rogueclothes/hats.dm]: you're right my bad!

**Review by Nakalon — APPROVED** [review]

**Comment by Nakalon** [comment]
Sorry for the delay love

**Pull request merged**